jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -259,6 +261,22 @@
                   (send bbox :header :frame_id)
                   (send bbox :pose)))
       bx))
+  (:get-movable-region-for-tote
+    (arm &key (offset (list 0 0 0)))
+    (let (cube)
+      (setq cube (send self :bbox->cube (gethash arm _order-bin-box)))
```

Comment:
This requires that `:recognize-order-bin-box` is called before it is called. You should avoid bugs like following:
```
(unless
   (setq order-bin-box (gethash arm _order-bin-box))
   (ros::ros-error "[:get-movable-region-for-tote] No data about order bin box. Call :recognize-order-bin-box first.")
   (return-from :get-movable-region-for-tote nil))
```